grpc-api/src/main/java/io/deephaven/grpc_api/arrow/ArrowFlightUtil.java

```diff
@@ -445,7 +448,9 @@ private synchronized void onExportResolved(final SessionState.ExportObject<Objec
                     updateIntervalMs = DEFAULT_UPDATE_INTERVAL_MS;
                 }
                 bmp = table.getResult(operationFactory.create(table, updateIntervalMs));
-                manage(bmp);
+                if (DynamicNode.notDynamicOrIsRefreshing(bmp)) {
```

Comment:
This can just be `bmp.isRefreshing()`; bmp is a `DynamicNode`, not some unknown object.